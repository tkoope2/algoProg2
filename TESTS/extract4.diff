9c9
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0>
11c11
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0>
13c13
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0>
15c15
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1>
17c17
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5>
19c19
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1>
21c21
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1>
23c23
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2>
25c25
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2>
27c27
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2>
29c29
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2 v2_3 v1_3 v0_3>
31c31
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2 v2_3 v1_3 v0_3 v5_3>
33c33
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2 v2_3 v1_3>
35c35
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4>
37c37
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4>
39c39
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4>
41c41
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5>
43c43
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5>
45c45
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0>
47c47
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1>
49c49
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2>
51c51
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2 v2_3>
53c53
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4>
55c55
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5>
57c57
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0>
59c59
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1>
61c61
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2>
63c63
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2 v2_3 v1_3 v0_3 v5_3 v4_3 v3_3>
65c65
< @     PATH: <>
---
> @     PATH: <v3_4>
67c67
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5>
69c69
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0>
71c71
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1>
73c73
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2>
75c75
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5 v3_5 v2_5 v1_5 v0_5 v5_5 v5_0 v4_0 v3_0 v2_0 v1_0 v0_0 v0_1 v5_1 v4_1 v3_1 v2_1 v1_1 v1_2 v0_2 v5_2 v4_2 v3_2 v2_2 v2_3 v1_3 v0_3 v5_3 v4_3>
77c77
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4>
79c79
< @     PATH: <>
---
> @     PATH: <v3_4 v2_4 v1_4 v0_4 v5_4 v4_4 v4_5>
