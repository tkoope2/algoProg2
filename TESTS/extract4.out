@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./dfs torus6x6.g v3_4 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@ v0_0
@     PATH: <>
@ v5_0
@     PATH: <>
@ v1_0
@     PATH: <>
@ v0_1
@     PATH: <>
@ v0_5
@     PATH: <>
@ v5_1
@     PATH: <>
@ v1_1
@     PATH: <>
@ v0_2
@     PATH: <>
@ v5_2
@     PATH: <>
@ v1_2
@     PATH: <>
@ v0_3
@     PATH: <>
@ v5_3
@     PATH: <>
@ v1_3
@     PATH: <>
@ v0_4
@     PATH: <>
@ v5_4
@     PATH: <>
@ v1_4
@     PATH: <>
@ v5_5
@     PATH: <>
@ v1_5
@     PATH: <>
@ v2_0
@     PATH: <>
@ v2_1
@     PATH: <>
@ v2_2
@     PATH: <>
@ v2_3
@     PATH: <>
@ v2_4
@     PATH: <>
@ v2_5
@     PATH: <>
@ v3_0
@     PATH: <>
@ v3_1
@     PATH: <>
@ v3_2
@     PATH: <>
@ v3_3
@     PATH: <>
@ v3_4
@     PATH: <>
@ v3_5
@     PATH: <>
@ v4_0
@     PATH: <>
@ v4_1
@     PATH: <>
@ v4_2
@     PATH: <>
@ v4_3
@     PATH: <>
@ v4_4
@     PATH: <>
@ v4_5
@     PATH: <>
